src/lqcontrol.jl

```diff
@@ -308,8 +308,8 @@ only for `min(ts_length, lq.capT)`
 represents `x_t`
 - `u_path::Matrix{Float64}` : A k x T matrix, where the t-th column represents
 `u_t`
-- `w_path::Matrix{Float64}` : A j x T+1 matrix, where the t-th column represents
-`lq.C*w_t`
+- `w_path::Matrix{Float64}` : A n x T+1 matrix, where the t-th column represents
+`lq.C*N(0,1)`
```

Comment:
Was the old doc string incorrect here? 

I can't recall, but I believe that putting `w_t` at the end should be right. If so, isn't that more specific than `N(0,1)` (i.e. We are saying which standard normal RV we have, not just an arbitrary one)